components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -621,6 +621,16 @@ define([
         },
 
         /**
+         * Returns a list of all available legends
+         */
+        getAllLegends: function() {
+            return [ this.getCancerLegend(),
+                     this.getHPOLegend(),
+                     this.getDisorderLegend(),
+                     this.getCarrierGeneLegend(), this.getRejectedGeneLegend(), this.getCandidateGeneLegend(), this.getCausalGeneLegend() ];
+        },
```

Comment:
Sure, it should be a separate thing. That's why I didn't request changes and only left a comment.